bpf/process/bpf_generic_retkprobe.c

```diff
@@ -28,7 +29,7 @@ struct bpf_map_def __attribute__((section("maps"), used)) config_map = {
 };
 
 __attribute__((section("kprobe/generic_retkprobe"), used)) int
-generic_kprobe_event(struct pt_regs *ctx)
+BPF_KRETPROBE(generic_kprobe_event, unsigned long rc_reg)
```

Comment:
could you please call it just 'ret', 'rc_reg' seems confusing to me that it has value of the register or something related

Reply:
Revised